signer/config_test.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestNodes(t *testing.T) {
+func Nodes(t *testing.T) {
```

Comment:
Seems wrong -- this will no longer be run as part of `go test`.

Reply:
good catch!